mint.json

```diff
@@ -158,6 +158,14 @@
         "monitor-data/monitors"
       ]
     },
+    {
+      "group": "Process data",
```

Comment:
Casing is inconsistent. I do prefer this style :-) want to switch the others to this, too? Did we settle on the guidelines?

Reply:
Nice catch! I've changed this to title case for now to align with the rest of the nav. Let's see how the wider conversation about capitalisation plays out.